StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1642SA1643CodeFixProvider.cs

```diff
@@ -53,13 +54,19 @@ public override FixAllProvider GetFixAllProvider()
                     continue;
                 }
 
-                var node = root.FindNode(diagnostic.Location.SourceSpan, findInsideTrivia: true) as XmlElementSyntax;
-                if (node == null)
+                var node = root.FindNode(diagnostic.Location.SourceSpan, findInsideTrivia: true) as XmlNodeSyntax;
+
+                var xmlElementSyntax = node as XmlElementSyntax;
+
+                if (xmlElementSyntax != null)
                 {
-                    continue;
+                    context.RegisterCodeFix(CodeAction.Create(DocumentationResources.SA1642SA1643CodeFix, token => GetTransformedDocumentAsync(context.Document, root, xmlElementSyntax), equivalenceKey: nameof(SA1642SA1643CodeFixProvider)), diagnostic);
+                }
+                else
+                {
+                    var xmlEmptyElementSyntax = node as XmlEmptyElementSyntax;
```

Comment:
:bulb: Don't assume the result of `as` is non-null. Either use a normal cast instead, or include a null check.